<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginScreen</class>
 <widget class="QMainWindow" name="LoginScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1385</width>
    <height>929</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#130f40;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>550</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>700</width>
        <height>550</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(100, 10, 164,0.5);
border: 0px;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>638</width>
         <height>77</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>117</width>
            <height>75</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>109</width>
            <height>75</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">image: url(:/kenkey1.png);
background-color: white;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>517</width>
            <height>75</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>517</width>
            <height>75</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(:/ad.jpg);
background-color: rgba(255, 165, 2,0.9);
border: 1px solid rgba(255, 56, 56,0.5);
border-radius: 0px;
</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;KENKEY BOUTIQUE&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>160</y>
         <width>634</width>
         <height>350</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>634</width>
         <height>350</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>634</width>
         <height>350</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #EA2027;
border: 52px solid rgba(234, 32, 39,0.5);
</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>290</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                           stop: 0 #4cd137, stop: 1#009432);
color: rgb(255, 255, 255);
selection-color: rgb(255, 0, 0);
border: 0px solid;
border-radius: 0px;
}
QPushButton:pressed {
    border: 1px solid #FFC312;
   color: #FFC312;
}

QPushButton:hover,
QPushButton:checked:hover{
    border: 1px solid #FFC312;
}</string>
        </property>
        <property name="text">
         <string>Register</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: none;
border:0px;
color: rgb(255, 255, 255);
</string>
        </property>
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="passwordEdit">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>160</y>
          <width>441</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit { 
    padding-top: 2px;     /* This QLineEdit fix  103, 111 */
    padding-bottom: 2px;  /* This QLineEdit fix  103, 111 */
    padding-left: 4px;
    padding-right: 4px;
    border-style: solid;
    background-color: rgb(255, 255, 255);
border: 2px solid rgba(0, 0, 0,1.0);
border-radius: 0px;
    color: rgb(0,0,0);
}

QLineEdit:disabled {
    background-color: #232629;
    color: #4D545B;
}

QLineEdit:hover{
    border: 2px solid rgba(60, 64, 198,1.0);
    color: rgb(0,0,0);
}

QLineEdit:selected{
   border: 2px solid rgba(60, 64, 198,1.0);
    color: rgb(0,0,0);
}</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QLineEdit" name="usernameEdit">
        <property name="geometry">
         <rect>
          <x>142</x>
          <y>90</y>
          <width>441</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit { 
    padding-top: 2px;     /* This QLineEdit fix  103, 111 */
    padding-bottom: 2px;  /* This QLineEdit fix  103, 111 */
    padding-left: 4px;
    padding-right: 4px;
    border-style: solid;
    background-color: rgb(255, 255, 255);
border: 2px solid rgba(0, 0, 0,1.0);
border-radius: 0px;
    color: rgb(0,0,0);
}

QLineEdit:disabled {
    background-color: #232629;
    color: #4D545B;
}

QLineEdit:hover{
    border: 2px solid rgba(60, 64, 198,1.0);
    color: rgb(0,0,0);
}

QLineEdit:selected{
   border: 2px solid rgba(60, 64, 198,1.0);
    color: rgb(0,0,0);
}</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>290</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                           stop: 0 #4cd137, stop: 1#009432);
color: rgb(255, 255, 255);
selection-color: rgb(255, 0, 0);
border: 0px solid;
border-radius: 0px;
}
QPushButton:pressed {
    border: 1px solid #FFC312;
   color: #FFC312;
}

QPushButton:hover,
QPushButton:checked:hover{
    border: 1px solid #FFC312;
}</string>
        </property>
        <property name="text">
         <string>Forgot password</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>170</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: none;
border:0px;
color: rgb(255, 255, 255);
</string>
        </property>
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>220</y>
          <width>93</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                           stop: 0 #434343, stop: 1#000000);
color: rgb(255, 255, 255);
selection-color: rgb(255, 0, 0);
border: 0px solid;
border-radius: 0px;
}
QPushButton:pressed {
    border: 1px solid rgba(0, 168, 255,1.0);
   color: rgba(0, 168, 255,1.0);
}

QPushButton:hover,
QPushButton:checked:hover{
    border: 1px solid rgba(0, 168, 255,1.0);
}
QPushButton:checked:disabled {
    background-color: #232629;
    border: 1px solid #31363B;
    color: #4D545B;
    border-radius: 4px;
    padding: 3px;
    outline: none;
}</string>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>228</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>109</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>340</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>303</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
